src/CoreBundle/Resources/config/form_types.xml

```diff
@@ -23,12 +23,12 @@
             <tag name="form.type" alias="sonata_type_datetime_range"/>
             <argument type="service" id="translator"/>
         </service>
-        <service id="sonata.core.form.type.date_picker" class="Sonata\CoreBundle\Form\Type\DatePickerType" public="true">
+        <service id="sonata.core.form.type.date_picker" class="Sonata\Form\Type\DatePickerType" public="true">
```

Comment:
I'm not sure if we should do this. If someone want's to use the old form type, there could be problems